# This is a message to hold data for found objects after the segmentation process.

Header header

Obj[] objects    		# array with an entry for each object, see also Obj.msg
